runtime/v2/runc/options/doc.go

```diff
@@ -15,3 +15,23 @@
 */
 
 package options
+
+import api "github.com/containerd/containerd/api/types/runc/options"
+
+// Options is a type alias of (github.com/containerd/containerd/api/types.runc/options).Options
+//
+// Deprecated: use github.com/containerd/containerd/api/types/runc/options.Options instead
```

Comment:
Can use doc links for these; pkg.go.dev will create a clickable link for those

```suggestion
// Deprecated: use [api.Options] instead
```